### ISSUE TITLE
Timeline filename doesn't show when Timeline is moved to the bottom Panel

### ISSUE THREAD

**Issue by r3m0t** [microsoft/vscode#166000]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 
- OS Version: 
- Version: 1.74.0-insider (user setup)
Commit: 9d5741f01a67beea273121615b2d015fc298d9ea
Date: 2022-11-10T05:21:13.252Z
Electron: 19.1.3
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19044
Sandboxed: Yes

Steps to Reproduce:

1. Correct behaviour (default layout, Timeline in Side Bar)
<img width="329" alt="image" src="https://user-images.githubusercontent.com/1485998/201077447-e3c06e5e-cbb1-41f8-8377-8a9afac722da.png">

3. Incorrect (not default layout, Timeline was dragged to Panel) - the filename is not shown , which is confusing when a timeline is pinned

<img width="1598" alt="image" src="https://user-images.githubusercontent.com/1485998/201077695-7fc48339-2d3c-41b2-a99e-91b638f77ecd.png">

**Comment by lramos15** [comment]
@daviddossett Are there any UX guidelines on how this type of thing is addressed when the view description is important but is unable to be present in the panel? I thought maybe we would show it on hover but we don't.

**Comment by gjsjohnmurray** [comment]
Also happens if you give Timeline a view container of its own in Sidebar.

Activity Bar at the top:

![image](https://github.com/microsoft/vscode/assets/6726799/320c25af-35c9-4e4c-922f-52c1efa5e817)

Activity Bar in default position:

![image](https://github.com/microsoft/vscode/assets/6726799/e06561a3-37bc-4ab3-a37c-8612fb18870d)

**Comment by gjsjohnmurray** [comment]
I have submitted a PR:

Default Activity Bar position:

![image](https://github.com/microsoft/vscode/assets/6726799/cfe98efd-2600-4cc3-bcb3-6b0a3f6334bc)

Top position:

![image](https://github.com/microsoft/vscode/assets/6726799/55323e26-b0ba-4478-a600-b405bc6927c1)

Timeline moved to Panel:

![image](https://github.com/microsoft/vscode/assets/6726799/76ea85e5-8023-4e2f-80e3-98eb5e873204)

**Comment by benibenj** [comment]
I don't think this was the right approach. Reverted due to [#214769](https://github.com/microsoft/vscode/issues/214769)

**Comment by benibenj** [comment]
Just a random idea. What about adding the file name as the first node in the list? You could even make it a tree and sticky scroll would always show the filename at the top then

**Comment by r3m0t** [comment]
@benibenj can this be reopened as it's still an issue? The commit message incorrectly closed it
